docs/pages/guides/server-components.mdx

```diff
@@ -0,0 +1,400 @@
+---
+title: Using React Server Components in Expo Router apps
+sidebar_title: React Server Components
+description: Learn about rendering React components on the server in Expo.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> Experimentally available in **SDK 52 and above**. This is a very early preview and not ready for production or wide use yet. This initial version is meant to help library authors understand universal React Server Components and add support in their libraries.
+
+Expo Router enables support for [React Server Components](https://react.dev/reference/rsc/server-components) on all platforms.
```

Comment:
```suggestion
```

Let's add this after the next block of text so that the flow is maintained. Right now it seems like there are two different ways the introduction is going to.